web-components/src/components/form/BridgeForm.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Field, Form, Formik, FormikErrors } from 'formik';
+
+import AgreeErpaCheckbox from '../inputs/AgreeErpaCheckbox';
+import AmountField from '../inputs/AmountField';
+import FormLabel from '../inputs/FormLabel';
+import SelectTextField from '../inputs/SelectTextField';
+import TextField from '../inputs/TextField';
+import {
+  requirementAgreement,
+  validateAmount,
+  validatePolygonAddress,
+} from '../inputs/validation';
+import { RegenModalProps } from '../modal';
+import Submit from './Submit';
+
+export interface BridgeProps {
+  batchDenom: string;
+  availableBridgeableAmount: number;
+  onSubmit: (values: FormValues) => Promise<void>;
+}
+
+interface FormProps extends BridgeProps {
+  onClose: RegenModalProps['onClose'];
+}
+
+export interface FormValues {
+  recipient: string;
+  amount?: number;
+  agreeErpa: boolean;
+}
+
+const BridgeForm = ({
```

Comment:
Not sure that this new form should belong to `web-components`. I know we have some already, but it feels like these kinds of components are more dedicated to the registry.
Our UI components should probably be agnostic to any Form library and without validation/business logic as well.

Curious to know what the rest of team thinks about this:
@wgwz @haveanicedavid @victordelval

Reply:
To keep the scope small here, I will leave the component in web-components for this PR, but will consider moving relevant pieces into registry modules in regen-network/regen-registry#1350